test/utils/helpFn.ts

```diff
@@ -21,6 +22,11 @@ export const MIN_RUNNING_TEST_BALANCE = 20000000000;
 export const TRANSFER_AMOUNT = 1000000000;
 export const RECEIVER_ADDRESS = '66fhJwYLiK87UDXYDQP9TfYdHpeEyMvQ3MK8Z6GgWAdyyCL3';
 const RECURRING_FREQUENCY = 3600;
+const ALITH_PRIVATE_KEY = '0x5fb92d6e98884f76de468fa3f6278f8807c48bebc13595d45af5bdc4da702133';
```

Comment:
Is this Alith from moonbeam? We should not expose any private key unless one is already exposed somewhere. Add a comment to link the github repo of moonbeam for this private key here.

Reply:
Fixed

Reply:
Yes, this looks good 👍